### PR TITLE
IPA-720 Add database model for task manager

### PR DESCRIPTION
**Pull request by shirleyqiantingliu**
# Problem / Feature
Add database model for task manager

# Changes
Added database model JobBatch and Job for task manager

### CODE REVIEW

File: src/db/20200128153514-IPA-720.main.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
```
Comment (huukias) [inline src/db/20200128153514-IPA-720.main.js]: This is a duplicate file?

File: src/utils/constants.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  /**
+   * Post actions for JobBatch.
+   * ALWAYS_DELETE: if we don't care the result at all, the job batch could be removed when it is finished
+   * DELETE_WHEN_SUCCEED: delete only when the job batch is finished suceesfully
+   * NONE: no post action. The result should be kept until intentionly deletion.
+   */
+  JOB_BATCH_POST_ACTION: {
+    ALWAYS_DELETE: 'alwaysDelete',
+    DELETE_WHEN_SUCCEED: 'deleteWhenSucceed',
+    NONE: 'None'
```
Comment (huukias) [inline src/utils/constants.js]: Nitpicking but I guess this should not be in capital letter as the others are not either

File: src/db/20200128153514-IPA-720.main.js (hunk 1 above)
Comment (shirleyqiantingliu) [reply src/db/20200128153514-IPA-720.main.js]: ?? What happened. I will remove it.

File: src/utils/constants.js (hunk 2 above)
Comment (shirleyqiantingliu) [reply src/utils/constants.js]: right.

File: src/db/models/Job.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+          });
+        }
+      },
+      timestamps: false,
+      tableName: "Job",
+      indexes: [
+        {
+          name: "JobUniqueIndex",
+          unique: true,
+          fields: ['JobBatchSN', 'ChunkId']
```
Comment (huukias) [inline src/db/models/Job.js]: Should these be jobBatchId and chunkId? (though I'm not sure if these indexes added to model are even used anywhere). At least classMethods uses jobBatchId as a reference

**Comment by huukias** [comment]
Not mandatory, but some comments about the columns could be useful in model files. Like to explain what Payload is used for

File: src/db/models/Job.js (hunk 3 above)
Comment (shirleyqiantingliu) [reply src/db/models/Job.js]: Since we might use sqlite, which require the fields here is database column not model fieldname.

**Pull request merged**